### PR TITLE
fix: Translate button sends users to homepage

### PR DESCRIPTION
**Pull request by O4FDev**
Fixes: #6287 

## Why 

As reported by @auroursa, the translate button on extended posts will send the user to the homepage unintentionally 
See behaviour here: https://bsky.app/profile/youareexisting.bsky.social/post/3lasuvhylqk2j

### CODE REVIEW

**Comment by gaearon** [comment]
where is that # coming from? we should remove it there instead.

**Comment by O4FDev** [comment]
> where is that # coming from? we should remove it there instead.

Ah, my bad - didn't realise there was already a way to handle this in there. 
Done

File: src/view/com/post-thread/PostThreadItem.tsx [hunk 1]
```diff
@@ -753,10 +753,9 @@ function ExpandedPostDetails({
             </Text>
 
             <InlineLinkText
-              to="#"
```
Comment (gaearon) [inline src/view/com/post-thread/PostThreadItem.tsx:756]: should this just be `to={translatorUrl}`? any reason not to do that?

Comment (O4FDev) [reply src/view/com/post-thread/PostThreadItem.tsx:756]: Good point! Will update

**Comment by auroursa** [comment]
I recently found that this issue isn’t limited to the web platform—it can occur across all platforms, including iOS. To be honest, it’s a bit annoying.

**Comment by auroursa** [comment]
@gaearon Sorry to bother you, could you take another look at this PR?

**Comment by O4FDev** [comment]
I believe there might've been an attempt to fix this - folks are reporting this as still an issue in 1.94 but more inconsistently. I think the version @gaearon came up with (implemented in this PR) might work the best, as I've not been able to replicate with it.

**Comment by surfdude29** [comment]
Just wondering, is this ready to merge? There's a message saying that this branch has conflicts 🤔

**Comment by O4FDev** [comment]
> Just wondering, is this ready to merge? There's a message saying that this branch has conflicts 🤔

I'll quickly fix the conflict so that it's not there but I suspect the solution will literally just be deleting what's there unfortunately, not sure if the maintainers can do a force merge w/ this version 🤷🏻

**Comment by O4FDev** [comment]
Closed based on @auroursa fix above being merged.

**Pull request closed without merging**